modules/analytics/src/views/full/index.tsx

```diff
@@ -265,21 +312,37 @@ const Analytics: FC<any> = ({ bp }) => {
 
   const getMetric = metricName => state.metrics.filter(x => x.metric === metricName)
 
-  const getTopItems = (metricName: string, type: string) => {
-    const grouped = _.groupBy(getMetric(metricName), 'subMetric')
-    const results = _.orderBy(
-      Object.keys(grouped).map(x => ({ name: x, count: _.sumBy(grouped[x], 'value') })),
-      x => x.count,
-      'desc'
-    )
+  const getTopItems = (options: {
+    metricName: string
+    type: string
+    nameRenderer?: (name: string) => string
+    filter?: (x: any) => boolean
+  }) => {
```

Comment:
Just being picky, but since the first two are required and standard, they can be left as an argument. When you need to add specific parameters, then you put them in an options field so it can be expanded easily if it needs more information

Reply:
Fixed in d4ea723